src/makeRobot.js

```diff
@@ -38,7 +38,68 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    goForward(length = 1) {
+      if (length <= 0) {
+        return this;
+      }
+
+      this.coords.y += length;
+
+      return this;
+    },
```

Comment:
```suggestion
    goForward(length = 1) {
      if (length > 0) {
        this.coords.y += length;
      }

      return this;
    },
```
lets simplify our methods